Philips.CodeAnalysis.MaintainabilityAnalyzers/NamespacePrefixAnalyzer.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Collections.Immutable;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Philips.CodeAnalysis.Common;
+
+namespace Philips.CodeAnalysis.MaintainabilityAnalyzers
+{
+	[DiagnosticAnalyzer(LanguageNames.CSharp)]
+	public class NamespacePrefixAnalyzer : DiagnosticAnalyzer
+	{
+		#region Non-Public Data Members
+
+		private const string Title = @"Namespace must use the predefined prefixes";
+		private const string MessageFormat = @"Namespace must use the predefined prefixes configured in the editorconfig file ie. [OrganizationName].[ProductName]";
```

Comment:
This message may want the detail of "namespace_prefix".  Otherwise, they won't know how to customize it.  Also, we should add your new analyzers to the README.MD table.

Reply:
Added a new rule to detect the absence of a namespace_prefix declaration in the .editorconfig file and updated the readme file with the new analyzers.